controls/srg_ctr/SRG-APP-000023-CTR-000055.yml

```diff
@@ -4,7 +4,107 @@ controls:
   - medium
   title: {{{ full_name }}} must use a centralized user management solution to
     support account management functions.
-  rules:
+  related_rules:
```

Comment:
Why did you move these rules to related? What is in your opinion missing from their checks that this control requies? btw maybe you were looking at an outdated version of the spreadsheet, these days we do allow anything by htpasswd

Reply:
I added one rule to related which is `ocp_no_ldap_insecure`

Reply:
oh and the rule should reference the control, too.

Reply:
Addressed.